src/helpers.ts

```diff
@@ -43,6 +50,13 @@ export function normalizeHex(s: string | undefined): string {
   }
 }
 
+function normalizeBigNumber(n: BigNumber): string {
+  if (n.gte(FLOAT_THRESHOLD)) {
```

Comment:
Now that I think of it, I like the version with try+toNumber even better because it'll also work for values <1 (and should still have no false positives).